File: qc-application/src/main/java/com/eager/questioncloud/application/authentication/AuthenticationToken.java
package com.eager.questioncloud.application.authentication;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class AuthenticationToken {
    private String accessToken;
    private String refreshToken;

    public static AuthenticationToken create(String accessToken, String refreshToken) {
        return new AuthenticationToken(accessToken, refreshToken);
    }
}

File: qc-api/qc-auth-api/src/main/kotlin/com/eager/questioncloud/authentication/model/AuthenticationToken.kt
package com.eager.questioncloud.authentication.model

class AuthenticationToken(
    val accessToken: String,
    val refreshToken: String
) {
    companion object {
        fun create(accessToken: String, refreshToken: String): AuthenticationToken {
            return AuthenticationToken(accessToken, refreshToken)
        }
    }
}
